src/csharp.tmLanguage.yml

```diff
@@ -2501,7 +2502,7 @@ repository:
           )*
         )
       )\s*
-      (?=\{|$)
+      (?=\{|//|/\*|$)
     captures:
```

Comment:
I'm not sure why this `new Type { ... }` syntax doesn't use begin-end with `#initializer-expression` like the other ones (see `anonymous-object-creation-expression` for example).

Reply:
I can only imagine the different hands which have touched this grammar. Feel free to make the switch, especially if it is easier to reason about.